zcash_client_backend/src/keys.rs

```diff
@@ -1,22 +1,34 @@
 //! Helper functions for managing light client key material.
-use zcash_address::unified::{self, Container, Encoding};
+use orchard;
+use zcash_address::unified::{self, Container, Encoding, Typecode};
 use zcash_primitives::{
-    consensus,
-    sapling::keys as sapling_keys,
+    consensus::{self},
```

Comment:
?
```suggestion
    consensus,
```

Reply:
I wonder why cargo fmt didn't fix this; it was left over from adding (and then removing) other types from the consensus crate.